examples/filter_streams_on_conditions/filter_streams_on_conditions.bal

```diff
@@ -64,7 +63,7 @@ function main (string... args) {
 }
 
 //Print the output events.
-function printStatusCount (StatusCount s) {
-    io:println("Event received; status: " + s.status +" and total occurrences: "+s.totalCount);
+function printStatusCount(StatusCount s) {
+    io:println("Event received; status : " + s.status + " and total occurrences : " + s.totalCount);
```

Comment:
It should be "status: ", there shouldn't be an extra space there. Please fix this and similar places.